apis/v1alpha1/ansibleRun_types.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+)
+
+// A ConfigurationSource represents the source of a AnsibleRun Configuration.
+// +kubebuilder:validation:Enum=Remote;Inline
+type ConfigurationSource string
+
+// Module sources.
+const (
+	ConfigurationSourceRemote ConfigurationSource = "Remote"
+	ConfigurationSourceInline ConfigurationSource = "Inline"
+)
+
+// AnsibleRunParameters are the configurable fields of a AnsibleRun.
+type AnsibleRunParameters struct {
```

Comment:
Assume AnsibleRun will also support vars overriding in this CR, right?

Reply:
Yes